esmvaltool/diag_scripts/ocean/diagnostic_maps_multimodel.py

```diff
@@ -340,13 +339,11 @@ def make_multiple_plots(cfg, metadata, obsname):
         path = diagtools.folder(cfg['plot_dir']) + plot_file
 
         # Saving file:
-        if cfg['write_plots']:
-            logger.info('Saving plots to %s', path)
-            plt.savefig(path, dpi=200)
+        logger.info('Saving plots to %s', path)
+        plt.savefig(path, dpi=200)
```

Comment:
it might be nice to use the function `esmvalcore.diag_scripts.shared.save_figure` here instead

Reply:
I really don't want to change other people's code any more than necessary without testing it, sorry.